src/sdk/pynni/requirements.txt

```diff
@@ -8,3 +8,6 @@ hyperopt==0.1.2
 
 # metis tuner
 scikit-learn==0.20
+
+# flops counter
+thop
```

Comment:
I don't think adding a dependency is a good idea. You can try importing `thop` in `counter.py`. If it doesn't exist, post a warning message telling users to install it.